MeetUpApp.Api/Program.cs

```diff
@@ -35,6 +41,7 @@
 
 builder.Services.AddEndpointsApiExplorer();
 builder.Services.AddSwaggerGen();
+builder.Services.AddFluentValidationRulesToSwagger();
```

Comment:
move all registrations of validation services to separate method

Reply:
Done